### ISSUE TITLE
question

### ISSUE THREAD

**Issue by mohdhaekal** [survivejs-translations/webpack-book-id#6]
Hello, sir
Is this project licensed ..

**Comment by bebraw** [comment]
Yup. [Here](https://creativecommons.org/licenses/by-nc-nd/3.0/). It's CC BY-NC-ND 3.0.

**Comment by mohdhaekal** [comment]
hello sir ... 
why do not we make in the form of file format. md
 its license

**Comment by bebraw** [comment]
Hi,

The [license is available online](https://creativecommons.org/licenses/by-nc-nd/3.0/). They maintain translated versions etc. of it there so I felt I don't have to bundle it with the book. Knowing the letter combination is enough to resolve it to a url.
